### PR TITLE
[Doc] Refactored doc files for better structure to allow gitbook integration

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This PR updates doc structure to be more suitable for gitbook integration. Once this PR is merged we can integrate with gitbook

## Gitbook documentation
Kepler.gl documentation is hosted on [gitbook](https://kepler-gl.gitbook.io/kepler-gl/). For more information [read here](https://docs.gitbook.com/)

### Documentation structure
The documentation layout is defined by __SUMMARY.md__ file where the table of contents define each entry has the following structure
```markdown
* [ENTRY_LABEL](FILE_PATH)
e.g.
* [Welcome](README.md)
``` 
The above file is used by Gitbook to generate the doc navigation visible on the left-hand side of Kepler.gl doc website. 
Gitbook also has the ability to show description for each folder/section of the documentation by creating an entry in __SUMMARY.md__
and create a new __README.md__ file within said folder. The README.md file is a Gitbook convention that treats README files as if they were the main entry file for each folder.

The following is an example of doc section in SUMMARY.md file:
```markdown
* [User guides](docs/user-guides/README.md)
``` 

### Update Documentation
The integration with Gitbook allows to update the documentation in two different ways:
- Update doc files in the Kepler.gl repo. Follow the PR flow like any other changes
- Update documentation directly on Gitbook.

For both scenarios, changes will be propagated from one system to the other and vice versa. When updating Gitbook, a new git commit will be push to the Kepler.gl master branch.

Signed-off-by: Giuseppe Macri <gmacri@uber.com>

### CODE REVIEW

File: bindings/kepler.gl-jupyter/js/lib/keplergl/components/panel-header.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
 import {PanelHeaderFactory, Icons} from 'kepler.gl/components';
 
 // TODO: Move the doc to public repo
-const KEPLER_DOC = 'https://github.com/keplergl/kepler.gl/blob/master/docs/keplergl-jupyter/user-guide.md';
+const KEPLER_DOC = 'https://github.com/keplergl/kepler.gl/blob/master/docs/keplergl-jupyter/README.md';
```
Comment (heshan0131) [inline bindings/kepler.gl-jupyter/js/lib/keplergl/components/panel-header.js:24]: should this be the gitbook link then?

File: src/constants/user-guides.js [hunk 2]
```diff
@@ -32,4 +32,4 @@ export const GUIDES_FILE_FORMAT =
 export const GITHUB_BUG_REPORT =
   'https://github.com/keplergl/kepler.gl/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBug%5D';
 export const GITHUB_USER_GUIDE =
-  'https://github.com/keplergl/kepler.gl/blob/master/docs/user-guides/a-introduction.md';
+  'https://github.com/keplergl/kepler.gl/blob/master/docs/user-guides/README.md';
```
Comment (heshan0131) [inline src/constants/user-guides.js:35]: gitbook?

File: website-gatsby/src/components/policy.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ import Header from './header';
 const GITHUB_PROJECT = 'https://github.com/keplergl/kepler.gl';
 const GITHUB_PROJECT_ISSUES = `${GITHUB_PROJECT}/issues`;
 const MAILING_LIST_URL = 'https://groups.google.com/d/forum/kepler-gl';
-const CONTRIBUTING_URL = 'https://github.com/keplergl/kepler.gl/blob/master/contributing/CONTRIBUTING.md';
+const CONTRIBUTING_URL = 'https://github.com/keplergl/kepler.gl/blob/master/contributing/README.md';
```
Comment (heshan0131) [inline website-gatsby/src/components/policy.js:30]: same here?

File: website-gatsby/src/content.js [hunk 4]
```diff
@@ -76,7 +76,7 @@ export const HEADER_NAVS = [
   },
   {
     text: 'Documentation',
-    link: 'https://github.com/keplergl/kepler.gl/blob/master/docs/api-reference/overview.md'
+    link: 'https://github.com/keplergl/kepler.gl/blob/master/docs/api-reference/README.md'
```
Comment (heshan0131) [inline website-gatsby/src/content.js:79]: and here?

File: website/src/components/policy.js [hunk 5]
```diff
@@ -28,7 +28,7 @@ const GITHUB_PROJECT = 'https://github.com/keplergl/kepler.gl';
 const GITHUB_PROJECT_ISSUES = `${GITHUB_PROJECT}/issues`;
 const MAILING_LIST_URL = 'https://groups.google.com/d/forum/kepler-gl';
 const CONTRIBUTING_URL =
-  'https://github.com/keplergl/kepler.gl/blob/master/contributing/CONTRIBUTING.md';
+  'https://github.com/keplergl/kepler.gl/blob/master/contributing/README.md';
```
Comment (heshan0131) [inline website/src/components/policy.js:31]: re

File: website/src/content.js [hunk 6]
```diff
@@ -74,7 +74,7 @@ export const HEADER_NAVS = [
   },
   {
     text: 'Documentation',
-    link: 'https://github.com/keplergl/kepler.gl/blob/master/docs/api-reference/overview.md'
+    link: 'https://github.com/keplergl/kepler.gl/blob/master/docs/api-reference/README.md'
```
Comment (heshan0131) [inline website/src/content.js:77]: re

**Comment by macrigiuseppe** [comment]
@heshan0131 all updated

**Pull request merged**